test/aws/osml/model_runner/tile_worker/test_variable_overlap_tiling_strategy.py

```diff
@@ -85,84 +115,90 @@ def test_compute_tiles(self):
                 ((3952, 3484), (4096, 4096)),
             ]
 
-        # A region on the right edge of the image
+        # Test region on the right edge of the image
         tiles = tiling_strategy.compute_tiles(((0, 20904), (4096, 8048)), tile_size, overlap)
         assert len(tiles) == 2
         for t in tiles:
             assert t in [((0, 20904), (4096, 4096)), ((3952, 20904), (4096, 4096))]
 
-        # A region on the bottom edge of the image
+        # Test region on the bottom edge of the image
         tiles = tiling_strategy.compute_tiles(((7904, 13936), (7580, 4096)), tile_size, overlap)
         assert len(tiles) == 2
         for t in tiles:
             assert t in [((7904, 13936), (4096, 4096)), ((7904, 17420), (4096, 4096))]
 
-        # The bottom right corner region
+        # Test bottom right corner region
         tiles = tiling_strategy.compute_tiles(((7904, 20904), (4096, 4096)), tile_size, overlap)
         assert len(tiles) == 1
         assert tiles[0] == ((7904, 20904), (4096, 4096))
 
     def test_compute_tiles_tiny_region(self):
+        """
+        Test that tiles are correctly computed for a small region that is smaller than the tile size.
+        """
         from aws.osml.model_runner.tile_worker import VariableOverlapTilingStrategy
 
         tiling_strategy = VariableOverlapTilingStrategy()
         overlap = (100, 100)
         tile_size = (4096, 4096)
 
-        # If the requested region is smaller than the actual tile size then the full region
-        # will be turned into a single tile. This is an odd edge case that we don't expect
-        # to see much but we don't want to error out and will instead just fall back to
-        # a partial tile strategy.
+        # Compute tiles for a small region
         tiles = tiling_strategy.compute_tiles(((10, 50), (1024, 2048)), tile_size, overlap)
+
+        # Verify that only a single tile is produced for the tiny region
         assert len(tiles) == 1
         assert tiles[0] == ((10, 50), (1024, 2048))
 
     def test_deconflict_features(self):
+        """
+        Test that duplicate features are properly deconflicted based on specified rules.
+        """
         from geojson import Feature
 
         from aws.osml.model_runner.inference import FeatureSelector
         from aws.osml.model_runner.tile_worker import VariableOverlapTilingStrategy
 
         tiling_strategy = VariableOverlapTilingStrategy()
 
+        # Define image and tiling parameters
         full_image_size = (25000, 12000)
         full_image_region = ((0, 0), full_image_size)
         nominal_region_size = (10000, 10000)
         overlap = (100, 100)
         tile_size = (4096, 4096)
 
+        # Sample feature inputs, including duplicates
         features = [
-            # Duplicate features in overlap of two regions, keep 1
```

Comment:
Please put these comments back. Each of these is a specific example.

Reply:
Ack will put them back in